gradio/routes.py

```diff
@@ -738,6 +738,21 @@ async def predict(
                     fn_index_inferred=fn_index_inferred,
                     root_path=root_path,
                 )
+                if (  # noqa: SIM102
```

Comment:
when I ran ruff, it didn't need this noqa statement? what does it do here?

Reply:
It raised an error because the two if statements can be combined into 1 but I think that would make the code harder to read.